src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -45,6 +45,7 @@ protected function configure()
                 new InputOption('domain', null, InputOption::VALUE_OPTIONAL, 'The messages domain'),
                 new InputOption('only-missing', null, InputOption::VALUE_NONE, 'Displays only missing messages'),
                 new InputOption('only-unused', null, InputOption::VALUE_NONE, 'Displays only unused messages'),
+                new InputOption('path', 'p', InputOption::VALUE_NONE, 'Treats bundle name as path'),
```

Comment:
Do we really need a new option? What about looking for a path if the bundle does not exist instead?